pcli/src/command/tx.rs

```diff
@@ -570,7 +553,6 @@ impl TxCmd {
                             .undelegate_claim(UndelegateClaimPlan {
                                 validator_identity,
                                 start_epoch_index,
-                                end_epoch_index,
                                 penalty,
```

Comment:
why do we have the client send the penalty at all? as opposed to just having the chain compute it

Reply:
because the penalty is an input to proof verification, and we want to be able to do all of the cryptographic verification up front, before doing any state accesses

Reply:
should the end epoch index then still be a part of the undelegate claim? as of this change, the client calculates the penalty assuming that the current epoch is >= the (chain enforced) end epoch

Reply:
i think it's fine to just include the penalty, since that's enough to verify the proofs, we can do some state access + computation to enforce that the supplied penalty is the correct one